client/src/data_layer_configs/lead_service_lines_by_water_systems_config.ts

```diff
@@ -114,22 +115,23 @@ const popupInfo: PopupInfo = {
       dataType: FeaturePropertyDataType.String,
       optional: true,
     },
-    {
-      label: 'Number of lead connections',
-      name: 'lead_connections_count',
-      dataType: FeaturePropertyDataType.Number,
-    },
-    {
-      label: 'Number of service lines',
-      name: 'service_connections_count',
-      dataType: FeaturePropertyDataType.Number,
-    },
     {
       label: 'Population served',
       name: 'population_served',
       dataType: FeaturePropertyDataType.Number,
     },
   ],
+  computedProperties: [
+    {
+      label: 'Percent lead lines',
+      name: 'percent_lead',
+      dataType: FeaturePropertyDataType.Percentage,
+      calculate: (features: GeoJsonProperties) =>
+        features == null
+          ? null
+          : features.lead_connections_count / features.service_connections_count,
```

Comment:
are these strings? does it cast to int when you divide?

Reply:
I don't think so, when I look at them they are raw numbers